### ISSUE TITLE
Get location from browser

### ISSUE THREAD

**Issue by JakeCooper** [AHAAAAAAA/PokemonGo-Map#164]
It would be fantastic if we had the option to get the location from the browser.

**Comment by CarlTaylor1989** [comment]
This for future development, this feature will get implemented.

**Comment by KonstantinKuklin** [comment]
+1

**Comment by volksport** [comment]
This is the killer feature for quick swapping between the app and this running on a personal server. I wouldn't ignore it for too long!

**Comment by Darkrift411** [comment]
I would really prefer a choice to either get current location, or to tap a spot on the map (long press? right click?) to select the option to "travel here". If a short distance, it could walk there. If longer than say a block, it could "drive" there (45-60mph). I am always worried about changing the script to a far off location so I normally do it in steps or stop it for the time I think it takes to get there. I understand this would be harder, but maybe implement in portions.

**Comment by Scyntrus** [comment]
There is actually a web API to change the location while the server is running, it looks like this:

localhost:5000/next_loc?lat=38.897660&lon=-77.036551

And the next time the server updates, it will move to that location.
However the larger your -st value, the longer it will take to move to the new location.

**Comment by Darkrift411** [comment]
Nice. I stopped browsing the code when so many people took this up or I might have noticed that. Thanks for the input. I was writing a bash script to start and stop this (before refresh was implemented) and it was going to read from a coords file that could be updated via a web form. As soon as I start to think of implementing something someone beats me to it so I just decided to make requests instead.
